src/interface/window.h

```diff
@@ -288,6 +289,7 @@ typedef struct rct_window {
 	sint8 var_4B8;
 	sint8 var_4B9;
 	uint8 colours[6];			// 0x4BA
+	uint8 visibility;			// VISIBILITY_CACHE
```

Comment:
Update the the window struct size comment if you are going to increase its size.